### DISCUSSION TITLE
Will this work on new 2024 Porsche Macan

### DISCUSSION THREAD

**Discussion by Dimitri2g** [Mr-MIBonk/M.I.B._More-Incredible-Bash#397] [General]
Will this work on the new 2024 Porsche Macan?
If it does do you happen to know where the SD card slot 1 is I can only find slot 2 under the armrest.

I think its PCM 5 so not even sure if this MIB solution will work. I have a 128 GB SD card (not micro sd) formatted fat32 and has the folder structure set up how you instructed. 

I can't seem to access the files from SD card slot 2 so I am in need of help.

Thanks in advance for any help.

Update 8/4/2024

Still have not gotten this working, I still don't know where the sd 1 card slot is. If I find something I will update here. I just don't know where it could be. 

I am hoping @Mr-MIBonk might be able to help or get someone that might know to help?

**Comment by iMccoyKitty** [comment]
2024 Macan +1.
The update menu under REM mode can not find the SD card 2 or USB device.
Need help...😭

**Comment by busschernick** [comment]
Did you ever find a solution for this one? On my 2020 Macan it also does not seem to read the SD card (inserts into the slot in the armrest). Did you try a USB device like a memory stick?

**Reply by Dimitri2g** [reply]
Nope, I been meaning to take a look into the glove box. From a diagram I saw online I think there might be an SD card slot in there. When I took a look it had a shielding or back plate of some kind on it so might need to remove that to expose the SD card slot.
Not sure yet but if I do I will report my findings.

**Comment by busschernick** [comment]
Thanks! I looked in the glovebox and pulled off a small panel but have not been able to find another SD card slot (maybe I did not look at the correct place). I tried the armrest SD card slot and a USB disk, both could not be selected/read in the menu unfortunately. Both were formatted in FAT32 so no idea what's going on, perhaps my radio version is just not compatible. 

2020 Macan (base)
MH2p_US_P0416_P2656
Firmware version: 2656
